lutris/gui/lutriswindow.py

```diff
@@ -439,8 +443,11 @@ def _open_browser(url):
 
     @GtkTemplate.Callback
     def on_resize(self, widget, *args):
-        """WTF is this doing?"""
-        self.window_size = widget.get_size()
+        """ size-allocate signal - updates stored window size and maximized state """
+        if not widget.get_window(): return
```

Comment:
Similar `if` clauses elsewhere in the codebase has the `return` on its own line. This should happen here too.